Kong/docs.konghq.com
# [DOCU-2921] Status endpoint + Kong health  check

New page + tutorial tempalte for KAG-76. Can decide if this is the right location when reviewing the draft content.

### Description

DOCU-2921

### Testing instructions

Netlify link: https://deploy-preview-5419--kongdocs.netlify.app/gateway/latest/production/monitoring/health-check/


### Checklist
